File: Components/Reporting/VB/DevExpress.DemoReports/RichText/Report.vb
Imports System
Imports System.ComponentModel
Imports System.Collections
Imports System.Diagnostics
Imports DevExpress.XtraReports.UI
Imports System.IO

Namespace XtraReportsDemos.RichText
    Partial Public Class Report
        Inherits DevExpress.XtraReports.UI.XtraReport

        Public Sub New()
            InitializeComponent()

            Name = ReportNames.RichText
            DisplayName = ReportNames.RichText
        End Sub
    End Class
End Namespace

File: Components/Reporting/CS/DevExpress.DemoReports/RichText/Report.cs
using System;
using System.ComponentModel;
using System.Collections;
using System.Diagnostics;
using DevExpress.XtraReports.UI;
using System.IO;

namespace XtraReportsDemos.RichText
{
	public partial class Report : DevExpress.XtraReports.UI.XtraReport
	{
		public Report()
		{
			InitializeComponent();

			Name = ReportNames.RichText;
            DisplayName = ReportNames.RichText;
		}
	}
}
